modules/35-blackbox-rightclick.yml

```diff
@@ -0,0 +1,10 @@
+name: blackbox-rightclick
+type: shell
+commands:
+  - mkdir -p /usr/share/nautilus-python/extensions/
+  - cp /sources/blackbox-rightclick/nautilus-open-in-blackbox-vos-1/nautilus-open-in-blackbox.py /usr/share/nautilus-python/extensions/
+
+source:
+  type: zip
+  url: https://github.com/fluoriteByte/nautilus-open-in-blackbox/archive/refs/tags/vos-1.zip
```

Comment:
I would prefer if we used a vanilla repo here. 
Would it be ok if we forked your fork?

Reply:
the original project already has my changes so you could just fork the original, its at https://github.com/ppvan/nautilus-open-in-blackbox/

Reply:
> I would prefer if we used a vanilla repo here.

Are you planning to use it as is without changing it?
I would love to see `Open in terminal` instead of `Open in BlackBox` and I believe it is much more clearer. But it is a matter of taste.

And how are you gonna install this extension? If in the future I need to change the terminal, I would also like to change the terminal that opens in nautilus, and it will be much harder to do with installing the extension system-wide

Reply:
Open in terminal is the clearer option, using generic names is the better option (as highlighted in the gnome hig), so that part shouldn't be changed

Reply:
> the original project already has my changes so you could just fork the original, its at https://github.com/ppvan/nautilus-open-in-blackbox/

in that case it'd be better to just use upstream instead of us forking it and having to maintain the fork